android/app/src/main/kotlin/coop/polypoly/polypod/updatenotification/UpdateNotification.kt

```diff
@@ -5,23 +5,65 @@ import coop.polypoly.polypod.Preferences
 import coop.polypoly.polypod.R
 
 class UpdateNotification(private val context: Context) {
-    val id = context.resources.getInteger(R.integer.update_notification_id)
+    companion object {
+        data class MockData(var id: Int? = null)
+        val mockData = MockData()
+    }
+
+    private enum class State {
+        NOT_SEEN,
+        PUSH_SEEN,
+        ALL_SEEN;
+
+        companion object {
+            fun parse(s: String?) =
+                s.let { name -> values().firstOrNull { it.name == name } }
+        }
+    }
+
+    val id = mockData.id
```

Comment:
OK but if notifications are defined by title plus text, with two notifications being identical if they have the same title plus text, would it be an alternative if you computed a hash from them? You wouldn't need the id, or to mock it...

Reply:
In theory yes - wanted to keep it simple I guess with the explicit ID. With the new logic where any notifications with an ID _higher_ than the previous one are shown, an integer ID makes more sense than it did before.